packages/react-composites/src/composites/CallWithChatComposite/styles/CallWithChatCompositeStyles.ts

```diff
@@ -43,3 +43,19 @@ export const ChatButtonContainerStyles: IStackStyles = {
     position: 'relative'
   }
 };
+
+/** @private */
+export const hiddenAutoFocusButtonStyles: IStackStyles = {
```

Comment:
I like this style to ensure there is nothing unexpected showing for customers

One more parameter to make it even safer opacity: 0;  ;-D